install-linux.sh

```diff
@@ -46,23 +46,36 @@ else
     SUDO='sudo -E'
 fi
 
-if [ $OS == "Debian" ]; then
+function get_platform() {
+    if ! command -v lsb_release &> /dev/null; then
+        NO_LSB=true
+    fi
+
+    if [ -n "${NO_LSB}" ]; then
+        echo "${DISTRIBUTION}"
+    else
+	lsb_release -sd
+    fi
+}
```

Comment:
No need for an extra flag here; can we just move the DISTRIBUTION up to line 51 and delete the middle `if` block on lines 54/55?

Reply:
Good call, `NO_LSB` used to get used elsewhere. 4f8e22c makes this change.